scrapy/pipelines/files.py

```diff
@@ -232,6 +233,7 @@ def from_settings(cls, settings):
         s3store = cls.STORE_SCHEMES['s3']
         s3store.AWS_ACCESS_KEY_ID = settings['AWS_ACCESS_KEY_ID']
         s3store.AWS_SECRET_ACCESS_KEY = settings['AWS_SECRET_ACCESS_KEY']
+        s3store.POLICY = settings['S3_STORE_ACL']
```

Comment:
Thanks for the heads up, @dangra - I was updating docs in the wrong place, should be fixed in 6137dd9. Files uploaded by feed exporters should already have ``private`` permissions - in this PR I want to make ``S3FilesStore`` use private ACL.